Description: Disable yargs test extends
 yargs-test-extends is not yet packaged
Forwarded: not-needed
Reviewed-By: Yadd <yadd@debian.org>
Last-Update: 2022-01-13

--- a/test/yargs.cjs
+++ b/test/yargs.cjs
@@ -1404,6 +1404,7 @@
         argv.z.should.equal(15);
       });
 
+/*
       // see: https://www.npmjs.com/package/yargs-test-extends
       it('allows a module to be extended, rather than a JSON file', () => {
         const argv = yargs()
@@ -1416,6 +1417,7 @@
         argv.a.should.equal(2);
         argv.c.should.equal(201);
       });
+*/
 
       it('ignores an extends key that does not look like a path or module', () => {
         const argv = yargs()
